eng/Install-SqlExpress.ps1

```diff
@@ -0,0 +1,24 @@
+$ErrorActionPreference = 'Stop';
+Set-PSDebug -Trace 1
+
+$packageName= 'sql-server-express'
+$url        = 'https://download.microsoft.com/download/8/4/c/84c6c430-e0f5-476d-bf43-eaaa222a72e0/SQLEXPR_x64_ENU.exe'
+$silentArgs = "/IACCEPTSQLSERVERLICENSETERMS /Q /ACTION=install /INSTANCEID=SQLEXPRESS /INSTANCENAME=SQLEXPRESS /UPDATEENABLED=FALSE"
+
+$tempDir = Join-Path (Get-Item $env:TEMP).FullName "$packageName"
+
+if (![System.IO.Directory]::Exists($tempDir)) { [System.IO.Directory]::CreateDirectory($tempDir) | Out-Null }
+$fileFullPath = "$tempDir\SQLEXPR.exe"
+
+(New-Object System.Net.WebClient).DownloadFile($url, $fileFullPath )
```

Comment:
Do we want to do something if the url for the download changes?

Reply:
The download.microsoft.com/download URL are very shelf stable.  As long as the product exists, so will this URL.

(It's also the URL the chocolatey installer uses)

Reply:
This will also throw some exception if it happens. :-)